check_command.go

```diff
@@ -48,33 +75,42 @@ func (c *CheckCommand) Run(request CheckRequest) ([]Version, error) {
 			continue
 		}
 
-		if release.TagName == nil {
-			continue
-		}
-		if _, err := version.NewVersionFromString(versionParser.parse(*release.TagName)); err != nil {
-			continue
+		if orderByTime {
+			// We won't do anything with the tags, so just make sure the filter matches the tag.
+			var tag string
+			if release.TagName != nil { tag = *release.TagName }
+			if !versionParser.re.MatchString(tag) { continue }
+			// Skip releases with zero time
+			if getTimestamp(release).IsZero() { continue }
```

Comment:
:question: When would this happen? Draft releases?

Reply:
I don't ever *expect* a zero (non-existent) timestamp, but this is here just to cover all the possible inputs the program could receive in a predictable way. Obviously, if a release doesn't have a timestamp, then its order by timestamp is undefined. Would you prefer if this printed a warning? I didn't bother with the warning precisely because the case is not really expected in a normal context of execution.

Reply:
I was mainly confused because to me the comment actually made it sound like something that is expected and needs to be handled. Adding a "(this should never happen)" or not handling at all would be fine with me.

Reply:
I improved the comment.